src/state.rs

```diff
@@ -226,19 +246,38 @@ impl NetworkState {
         A: Canon,
         R: Canon,
     {
+        let _span = trace_span!(
+            "outer transact",
+            block_height = ?block_height,
+            target = ?target,
+            gas_limit = ?gas_meter.limit(),
+        );
+
         // Fork the current network's state
         let mut fork = self.clone();
 
-        // Use the forked state to execute the transaction
+        // Use the forked state to execute the transaction.
```

Comment:
No need for the period at the end of single line comment (otherwise for consistency we should do that for line 256 and all other comments.

Reply:
Fixed!